http/server.ts

```diff
@@ -371,21 +374,32 @@ export class Server {
           error instanceof Deno.errors.ConnectionReset ||
           error instanceof Deno.errors.NotConnected
         ) {
-          // Backoff after transient errors to allow time for the system to
-          // recover, and avoid blocking up the event loop with a continuously
-          // running loop.
-          if (!acceptBackoffDelay) {
-            acceptBackoffDelay = INITIAL_ACCEPT_BACKOFF_DELAY;
-          } else {
-            acceptBackoffDelay *= 2;
-          }
-
-          if (acceptBackoffDelay >= MAX_ACCEPT_BACKOFF_DELAY) {
-            acceptBackoffDelay = MAX_ACCEPT_BACKOFF_DELAY;
+          if (!this.#closed) {
```

Comment:
I think this change can be reverted with the addition of the abort controller and my previous comment about moving the `abort()` down.

Reply:
Aha, that makes sense.

Reply:
Made the change here: https://github.com/denoland/deno_std/pull/2732/commits/e65d8bc4ad5627ebb8a4e504320784ff1293d390